susemanager/src/mgr_bootstrap_data.py

```diff
@@ -202,6 +202,10 @@
     "python-setuptools",
 ]
 
+ENHANCE12SP5 = [
+    "dbus-1-glib",
```

Comment:
Is this package new on SP5? I think not.
Would it hurt when we copy it always?

Reply:
@rjmateus when you install this package on JeOS, does it install other packages as well?
We need to list all the dependencies of this one.

Reply:
> Is this package new on SP5? I think not.

No, not new. But on SP5 not JeOS and on previous SLE12 versions (JeOS and not JeOS) it is installed by default as part of the OS. It's a requirement of `dbus-1-python` which is listed at `PKGLIST12`.

So this is only for JeOS SLE12SP5.

> Would it hurt when we copy it always?

Only in terms of diskspace, FMPOV. If the package is installed on the client, our version will not get installed (for other SPs and 12SP5 non-JeOS).

So if you are happy with having it at all bootstrap repos duplicated, I can add it to `PKGLIST12`, of course :-)

Reply:
As discussed on rocket.chat, let's keep it simple.